### DISCUSSION TITLE
Enforcing only TLS 1.3 ciphers in the browser will result in a permanent rejection and an error message when attempting to upload files on GitHub

### DISCUSSION THREAD

**Discussion by Nurmagoz** [community/community#60446] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

I'm using only TLS 1.3 ciphers on my browser for better security and speed. Browsing GitHub and submitting tickets works fine, except for one issue: uploading files. It appears that there are limited ciphers accepted, and they haven't been upgraded to TLS 1.3, despite it being available for several years.

### Steps to reproduce

- Install Firefox
- Go to about:config and search for `security.tls.version.min` and then change it from `3` to `4`
- Login to github and try to upload any file
- There you go..

### Screenshot for the issue

![githubbug](https://github.com/community/community/assets/11895339/af206cf3-d2b8-4920-9a82-4688902ce874)

Firefox console (It looks like amazon has this ciphers limitation which github upload rely on):

![githubbug2](https://github.com/community/community/assets/11895339/d2c0e850-1670-42fd-91cf-f45ab2b057f1)

Thx!

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @TNTBOMBOM ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by Nurmagoz** [comment]
It seems that GitHub doesn't have the authority to make the decision to transition for better security.

**Comment by Jaber-Saed** [comment]
Hello,

Thank you for bringing this issue to our attention regarding TLS 1.3 ciphers and their impact on file uploads on GitHub. It's crucial that security and compatibility are maintained across all aspects of a platform, and your detailed explanation helps shed light on the problem.

I understand that you've observed that while browsing and submitting tickets on GitHub using TLS 1.3 ciphers works fine, there's an issue when it comes to uploading files. It seems that the ciphers accepted during the file upload process are limited and have not been upgraded to support TLS 1.3, even though it has been available for a substantial period of time.

To reproduce the issue, you've outlined the steps, which include configuring Firefox to use TLS 1.3 ciphers and then attempting to upload a file on GitHub. The screenshots you've provided, along with the console output from Firefox, highlight the issue and the specific ciphers that are creating a limitation for the upload process.

This limitation might be related to the specific ciphers that GitHub relies on during file uploads, which could be provided by external services like Amazon. It's important that the ciphers used during uploads are both secure and compatible to ensure smooth and secure data transfer.

I recommend reaching out to GitHub's support team directly with this information. GitHub's support team would be best equipped to investigate the issue, provide insights, and potentially work on addressing the problem in collaboration with the relevant parties. Your feedback helps in improving the platform and ensuring that security and functionality are maintained.

Thank you for following the guidelines and providing a detailed explanation of the issue you've encountered. Your contribution in highlighting this issue is valuable to the GitHub community and the platform's ongoing development.

Best regards,
Jaber

**Comment by Nurmagoz** [comment]
![fungithub](https://github.com/community/community/assets/11895339/3c9e395d-cc05-4b1a-9f39-95f8a654e78a)

![fungithub2](https://github.com/community/community/assets/11895339/0c7dd251-be0e-4b7b-9e4b-4caec7adb3f5)

https://support.github.com/ticket/personal/0/2306004

What a LOL reply (and closed the ticket without even noticing my issue already exist in the community discussion which is mentioned in my opening ticket omg..).

Microsoft's support and products are consistently characterized by a lack of real support and hardened security measures.

**Comment by Nurmagoz** [comment]
Still the same BS, github cant support TLS 1.3 for files upload..
